src/popup/App.vue

```diff
@@ -222,6 +239,10 @@ export default defineComponent({
     box-shadow: variables.$color-border 0 0 0 1px;
     transform: translate(0, 0); // Create custom viewport for fixed elements
 
+    @include mixins.desktop {
+      border-radius: variables.$border-radius-app;
```

Comment:
The `#app` already has the border-radius. Is this necessary to add another one? And if yes I suggest to use the CSS variable `--screen-border-radius` or just inherit it.